pax/plugins/io/WaveformSimulator.py

```diff
@@ -191,12 +227,27 @@ def s2_after_pulses(self):
                 Y[electron_id]
                 )
             # queue the photons caused by the s2 after pulses
-            self.simulator.queue_signal(
+            arriving_photon_times_top, arriving_photon_times_bottom =  self.simulator.queue_signal(
                 s2_ap_photon_times,
                 X[electron_id],
                 Y[electron_id],
                 -self.config['gate_to_anode_distance']
                 )
+            peak_top_fraction = float(len(arriving_photon_times_top)) / float(
+                len(arriving_photon_times_bottom) + len(arriving_photon_times_top) )
+            self.store_true_peak('photoionization_afterpulse',
+                                          g4_id,
+                                          t = s2_ap_electron_time,
+                                          x = X[electron_id],
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent
* [E251] unexpected spaces around keyword / parameter equals

<!-- Comment by [SideCI](https://sideci.com) -->